lib/fortnox/api/models/base.rb

```diff
@@ -30,6 +30,17 @@ def self.new( hash )
           IceNine.deep_freeze( obj )
         end
 
+        # This filtering logic could be improved since it is currently O(N*M).
+        def attributes( *options )
+          return self.class.schema if options.nil?
+
+          options = Array(options)
```

Comment:
Maybe choose a different variable name here instead of mutating options?

Reply:
Maybe, but on the other hand I'm just ensuring that the argument is an array, I'm not really mutating it. Also I think this is unnecessary, the argument should already be an array from how I get them. I'll verify.